server/jms/src/main/java/org/hawkular/apm/server/jms/CommunicationDetailsCacheMDB.java

```diff
@@ -67,7 +68,8 @@ public void init() {
         setProcessor(new AbstractProcessor<CommunicationDetails,Void>(ProcessorType.ManyToMany) {
 
             @Override
-            public List<Void> processManyToMany(String tenantId, List<CommunicationDetails> items) throws Exception {
+            public List<Void> processManyToMany(String tenantId, List<CommunicationDetails> items)
+                    throws RetryAttemptException {
                 communicationDetailsCache.store(tenantId, items);
```

Comment:
Correct - however I think the current cache API needs to define a checked exception - I'll create a separate jira for this.